index.html

```diff
@@ -9,9 +9,11 @@
     <body>
         <div id="searchBox" class="text-center">
             <h1>Dictionary</h1><br>
-            <input class="form-control" placeholder="Add your word" id="userInput" onfocus=this.value='' autocomplete="on"/>
-            <button id = "addButton" class="btn btn-danger" onclick="addWords()">Add word</button><br>
-            <input class="form-control" placeholder="Search your word" onkeyup="showResults(value)">
-            <div id="result"></div>
+            <form action="javascript:addWords(this)">
+                <input id="input"class="form-control" placeholder="Add your word" onfocus=this.value='' autocomplete="on"/>
```

Comment:
Here `onfocus=this.value=''` I would change to reset the value after the word was added